core/modules/qproc/QuerySession.cc

```diff
@@ -66,24 +66,23 @@
 #include "query/SelectStmt.h"
 #include "query/SelectList.h"
 #include "query/typedefs.h"
+#include "util/IterableFormatter.h"
 
 #define DEBUG 0
```

Comment:
Sorry for being unclear. The question was actually about `DEBUG` macro. I know it's not part of your commit but maybe you can look at it if you touch this file anyway.